spec/features/user_views_all_books_page.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'User visits books page' do
+  it 'shows all books' do
+    create :book, title: 'Book 1'
+    create :book, title: 'Book 2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.